keras_nlp/tokenizers/word_piece_tokenizer_trainer_test.py

```diff
@@ -32,18 +32,19 @@ def test_dataset_input(self):
 
     def test_filenames_input(self):
         test_text = "baa maa caa saa aaa"
-        with open("test.txt", "w+") as f:
+        with open(os.path.join(self.get_temp_dir(), "test.txt"), "w+") as f:
```

Comment:
for this and below, just use a local variable

vocab_file = os.path.join(self.get_temp_dir(), "test.txt")

then reference that each place it is used

Reply:
do this for in each unit test if that's unclear, but the code will read a little better